amy/templates/emails/scheduled_email_reschedule.html

```diff
@@ -13,6 +13,12 @@ <h1>
 
 {% include "emails/scheduled_email_header_details.html" with scheduled_email=scheduled_email %}
 
+{% if scheduled_email.state == 'cancelled' %}
+<div class="alert alert-warning" role="alert">
+  <strong>Warning</strong>: if you reschedule a cancelled email, it will become scheduled again and may be sent immediately.
```

Comment:
```suggestion
  <strong>Warning</strong>: if you reschedule a cancelled email, it will become scheduled again and will be sent immediately if the scheduled time is in the past.
```

Reply:
I'll use your suggestion, but it doesn't capture exactly what I had in my mind. The email can be sent (almost) immediately if the scheduled time is in the past or very close to current time.

Reply:
Makes sense. I was thinking only about capturing the part of the behaviour that might be unexpected